website/content/en/docs/olm-integration/quickstart-bundle.md

```diff
@@ -0,0 +1,152 @@
+INFO[0001] Fetching resources for version "latest"
+INFO[0007] Creating CRDs and resources
+INFO[0007]   Creating CustomResourceDefinition "clusterserviceversions.operators.coreos.com"
+INFO[0007]   Creating CustomResourceDefinition "installplans.operators.coreos.com"
+INFO[0007]   Creating CustomResourceDefinition "subscriptions.operators.coreos.com"
+...
+NAME                                            NAMESPACE    KIND                        STATUS
+clusterserviceversions.operators.coreos.com                  CustomResourceDefinition    Installed
+installplans.operators.coreos.com                            CustomResourceDefinition    Installed
+subscriptions.operators.coreos.com                           CustomResourceDefinition    Installed
+catalogsources.operators.coreos.com                          CustomResourceDefinition    Installed
+...
+```
+
+**Note:** By default, `olm status` and `olm uninstall` auto-detect the OLM version installed in your cluster.
+This can fail if the installation is broken in some way, so the version of OLM can be overridden using the
+`--version` flag provided with these commands.
+
+## Creating a bundle
+
+_If working with package manifests, see the [package manifests quickstart][quickstart-package-manifests]._
+
+We will now create bundle manifests by running `make bundle` in the root of the memcached-operator project.
+
+```console
+$ make bundle
+/home/user/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+operator-sdk generate kustomize manifests -q
+kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version 0.0.1  
+INFO[0000] Building annotations.yaml                    
+INFO[0000] Writing annotations.yaml in /home/user/go/src/github.com/test-org/memcached-operator/bundle/metadata
+INFO[0000] Building Dockerfile                          
+INFO[0000] Writing bundle.Dockerfile in /home/user/go/src/github.com/test-org/memcached-operator
+operator-sdk bundle validate ./bundle
+INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
+INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
+INFO[0000] All validation tests have completed successfully
+```
+
+A bundle manifests directory `bundle/manifests` containing a CSV and all CRDs in `config/crds`, a bundle
+[metadata][bundle-metadata] directory `bundle/metadata`, and a [Dockerfile][bundle-dockerfile] `bundle.Dockerfile`
+have been created in the Operator project. These files have been statically validated by
+`operator-sdk bundle validate` to ensure the on-disk bundle representation is correct.
+
+## Deploying an Operator with OLM
+
+At this point in development we've generated all files necessary to build the memcached-operator bundle.
+Now we're ready to test and deploy the Operator with OLM.
+
+### Testing bundles
+
+Coming soon.
```

Comment:
Add a TODO comment and tag @jmccormick2001, @jmrodri, and @rashmigottipati to link to and/or document `scorecard` and `run bundle` when those commands are ready.